src/features/donations/components/PaymentFunctions.ts

```diff
@@ -271,13 +271,15 @@ export async function payDonationFunction({
       if (paidDonation.status === 'failed') {
         setIsPaymentProcessing(false);
         setPaymentError(paidDonation.message);
-      } else if (paidDonation.paymentStatus === 'success' || paidDonation.paymentStatus === 'pending' ||
-      paidDonation.data.status === "success" || 
-      paidDonation.data.status === "paid" ||
-      paidDonation.data.paymentStatus === "paid") {
+        return paidDonation;
+      } else if (paidDonation.paymentStatus === 'success'
+        || paidDonation.paymentStatus === 'pending'
+        || paidDonation.paymentStatus === 'paid'
+        || paidDonation.status === 'success'
+        || paidDonation.status === 'pending'
```

Comment:
@harshvitra I added this just to be sure as `paidDonation.paymentStatus === 'pending'` was also requested before -but I am not sure about the status model currently used for payments.

Reply:
Credit card payment on staging (not development) backend are getting finalized now, Paypal not.

Reply:
This is prob because of missing web hooks to staging from PayPal.